src/java/voldemort/store/readonly/swapper/AdminStoreSwapper.java

```diff
@@ -146,6 +151,10 @@ public void invokeRollback(final String storeName, final long pushVersion) {
     public Map<Node, Response> invokeFetch(final String storeName,
                                            final String basePath,
                                            final long pushVersion) {
+        // This should result in 5 minutes max of wait time, for failure scenarios which are potentially recoverable.
```

Comment:
Is 5 mins enough for normal upgrade of one node? Or do you think it is worth creating a plugin config for it considering sometimes the deployment for remote colo could be slow.